src/main/webapp/src/components/FeedbackForm.vue

```diff
@@ -0,0 +1,220 @@
+<template>
+    <div>
+        <div data-testid="confirmModalContent" class="feedback-tip-text">
+            Die Inhalte werden veröffentlicht sein.
+        </div>
+        <div class="feedback-questions-container">
+            <div class="row">
+                <div class="column">
+                    <label
+                        for="feedback-participant-name"
+                        class="feedback-label"
+                        >Wie lautet Dein Name?</label
+                    >
+                    <input
+                        type="text"
+                        v-model="$v.feedback.participantName.$model"
+                        @change="$emit('feedback', feedback)"
+                        id="feedback-participant-name"
+                        placeholder="Kurzantwort-Text (Optional)"
+                    />
+                    <span
+                        v-if="$v.feedback.participantName.$error"
+                        class="feedback-validation-text"
+                        >Die maximale Länge sind 255 Zeichen.
+                    </span>
+                </div>
+            </div>
+            <div class="row">
+                <div class="column">
+                    <label for="feedback-dislikes" class="feedback-label"
+                        >Was hat Dir im Rahmen der Fortbildung nicht
+                        gefallen?</label
+                    >
```

Comment:
I do not get why you find the formatting weird ? Prettier does format the file automatically on save. In addition there is no lint errors.

Reply:
Just because Prettier doesn't throw an error doesn't necessarily mean the formatting is good. The body of an HTML element just shouldn't be on the same line as the tags